cmake/external/CudaRdcUtils.cmake

```diff
@@ -460,6 +528,15 @@ function(cuda_rdc_add_library target)
     EXPORT_PROPERTIES "CUDA_RUNTIME_LIBRARY;CUDA_RDC_LIBRARY_TYPE;CUDA_RDC_FINAL_LIBRARY;CUDA_RDC_MIDDLE_LIBRARY;CUDA_RDC_STATIC_LIBRARY"
   )
 
+  if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
```

Comment:
Are you sure this only applies to linux? Wouldn't it also technically depend on the linker (ld versus gold vs ...)?

Reply:
Removed.